media/css/protocol/components/_footer.scss

```diff
@@ -27,6 +27,11 @@ $image-path: '/media/protocol/img';
             &.spotify {
                 background-image: url('/media/img/logos/social/spotify-white.svg');
             }
+
+            &.mastodon {
+                background-image: url('/media/img/logos/social/mastodon-white.svg');
```

Comment:
Also needs the black version in the light themed footer used on Firefox pages (scroll further down)